File: docker/src/jentic_scorecard_runner/exit_codes.py
"""Container exit codes per architecture doc §6."""

from enum import IntEnum


class ExitCode(IntEnum):
    SUCCESS = 0
    GENERIC_ERROR = 1
    AUTH_INVALID_KEY = 2
    GATE_REJECTED = 3
    SPEC_FAILURE = 5
    ENGINE_FAILURE = 6

File: packages/cli/src/exit-codes.ts
export const ExitCode = {
  SUCCESS: 0,
  GENERIC_ERROR: 1,
  AUTH_INVALID_KEY: 2,
  GATE_REJECTED: 3,
  DOCKER_MISSING: 4,
  SPEC_FAILURE: 5,
  ENGINE_FAILURE: 6,
  RATE_LIMITED: 7,
} as const;

export type ExitCode = (typeof ExitCode)[keyof typeof ExitCode];
